components/raftstore-v2/src/operation/command/admin/conf_change.rs

```diff
@@ -196,6 +196,9 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
             .lock()
             .unwrap()
             .set_region(self.region(), true, &self.logger);
+        // Update leader's peer list after conf change.
+        self.read_progress()
```

Comment:
Could you check other place where `set_region` is called but not does not call `update_leader_info`?

Reply:
I've checked `on_apply_res_split` and `on_apply_res_commit_merge`. both have called `update_leader_info` in `set_region`. can we directly call set_region here?